app/src/main/kotlin/dev/patrickgold/florisboard/app/settings/advanced/BackupScreen.kt

```diff
@@ -253,6 +316,53 @@ internal fun BackupFilesSelector(
             checked = filesSelector.imeTheme,
             text = stringRes(R.string.backup_and_restore__back_up__files_ime_theme),
         )
+
+        CheckboxListItem(
+            onClick = {
+                if (!filesSelector.clipboardData) {
+                    filesSelector.clipboardTextItems = true
+                    filesSelector.clipboardImageItems = true
+                    filesSelector.clipboardVideoItems = true
+                } else {
+                    filesSelector.clipboardTextItems = false
+                    filesSelector.clipboardImageItems = false
+                    filesSelector.clipboardVideoItems = false
+                }
+                filesSelector.clipboardData = filesSelector.validateClipboardCheckbox()
+            },
+            checked = filesSelector.clipboardTextItems && filesSelector.clipboardImageItems && filesSelector.clipboardVideoItems,
+            text = stringRes(R.string.backup_and_restore__back_up__files_clipboard_history)
+        )
+
+
+        CheckboxListItem(
+            onClick = {
+                filesSelector.clipboardTextItems = !filesSelector.clipboardTextItems
+                filesSelector.clipboardData = filesSelector.validateClipboardCheckbox()
+            },
+            checked = filesSelector.clipboardTextItems,
+            text = stringRes(R.string.backup_and_restore__back_up__files_clipboard_history__clipboard_text_items),
+            isSecondaryListItem = true,
+        )
+        CheckboxListItem(
+            onClick = {
+                filesSelector.clipboardImageItems = !filesSelector.clipboardImageItems
+                filesSelector.clipboardData = filesSelector.validateClipboardCheckbox()
+            },
+            checked = filesSelector.clipboardImageItems,
+            text = stringRes(R.string.backup_and_restore__back_up__files_clipboard_history__clipboard_text_items),
+            isSecondaryListItem = true,
+        )
+        CheckboxListItem(
+            onClick = {
+                filesSelector.clipboardVideoItems = !filesSelector.clipboardVideoItems
+                filesSelector.clipboardData = filesSelector.validateClipboardCheckbox()
+            },
+            checked = filesSelector.clipboardVideoItems,
+            text = stringRes(R.string.backup_and_restore__back_up__files_clipboard_history__clipboard_text_items),
```

Comment:
```suggestion
            text = stringRes(R.string.backup_and_restore__back_up__files_clipboard_history__clipboard_video_items),
```

(you used the incorrect string, probably due to copy/paste)